core/interop/pinvoke/marshaling/readme.md

```diff
@@ -1,26 +1,35 @@
-.NET Core PInvoke Marshaling Sample
-================
+---
```

Comment:
I believe it's more common to have the file name as "README.md" (README is all uppercase).